recipes/cmlgenerator/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "cmlgenerator" %}
+{% set version = "2.0.0" %}
+{% set sha256 = "64e075430c9d4ec3d4458881c874762dad3b4021f024614da1578217f764e88d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.developers.cam.ac.uk/ch/hunter/ssiptools/mepsgen/cmlgenerator/-/archive/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256:  {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script_env:
+    - SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools
+    - setuptools_scm
+  run:
+    - python >=3.6
+    - numpy
```

Comment:
Upstream also lists `lxml`. Any reason this doesn't?

Reply:
Thanks for looking at this. Yes, I think you're correct. I will add this in. Many thanks for the spotting this.